frontend/admin/src/js/components/user/UserProfileDocument.tsx

```diff
@@ -0,0 +1,222 @@
+  ThumbUpIcon,
+} from "@heroicons/react/outline";
+import { useIntl } from "react-intl";
+import LanguageInformationSection from "@common/components/UserProfile/ProfileSections/LanguageInformationSection";
+import GovernmentInformationSection from "@common/components/UserProfile/ProfileSections/GovernmentInformationSection";
+import WorkLocationSection from "@common/components/UserProfile/ProfileSections/WorkLocationSection";
+import WorkPreferencesSection from "@common/components/UserProfile/ProfileSections/WorkPreferencesSection";
+import DiversityEquityInclusionSection from "@common/components/UserProfile/ProfileSections/DiversityEquityInclusionSection";
+import RoleSalarySection from "@common/components/UserProfile/ProfileSections/RoleSalarySection";
+import { notEmpty } from "@common/helpers/util";
+import ExperienceByTypeListing from "@common/components/UserProfile/ExperienceByTypeListing";
+import { Applicant } from "../../api/generated";
+import AdminAboutSection from "./AdminAboutSection";
+
+const HeadingWrapper: React.FC = ({ children }) => {
+  return (
+    <div style={{ display: "flex", alignItems: "baseline" }}>{children}</div>
+  );
+};
+
+export interface HeadingProps {
+  as?: "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
+  icon?: React.FC<{ className: string }>;
+}
+
+const Heading: React.FC<HeadingProps & HTMLAttributes<HTMLHeadingElement>> = ({
+  icon,
+  children,
+  as = "h2",
+  ...rest
+}) => {
+  const El = as;
+  const Icon = icon || null;
+
+  return (
+    <El
+      data-h2-display="b(flex)"
+      data-h2-font-weight="b(800)"
+      data-h2-align-items="b(center)"
+      data-h2-margin="b(top, none) b(bottom, m)"
+      data-h2-justify-content="b(start)"
+      {...rest}
+    >
+      {Icon && <Icon className="heading-icon" />}
+      <span>{children}</span>
+    </El>
+  );
+};
+
+export interface UserProfileDocumentProps {
+  applicant: Applicant;
+}
+
+export const UserProfileDocument = React.forwardRef<
+  HTMLDivElement,
+  UserProfileDocumentProps
+>(({ applicant }, ref) => {
+  const intl = useIntl();
+  return (
+    <div
+      style={{
+        display: "none",
+      }}
+    >
+      {/* a bunch of styling to tweak components for printing */}
+      <style type="text/css" media="print">{`
+
+        html{font-size:75%}
+
+        .heading-icon {
+          flex-shrink: 0;
+          height: 1.5rem;
+          width: 1.5rem;
+          margin-right: 1rem;
+        }
+
+        .accordion-header {
+          width: 100%;
+          text-align: left;
+          padding-bottom: 1rem;
+          padding-top: 1rem;
+          padding-right: 1.5rem;
+          padding-left: 1rem;
+        }
+
+        .accordion-header-context {
+          display: flex;
+          flex-shrink: 0;
+          flex: initial;
+          max-width: 100%;
+          align-items: center;
+          flex-direction: row;
+        }
+      }
+    `}</style>
+      <div ref={ref}>
+        <div className="print-container">
```

Comment:
All this is copied from the UserProfile component to ensure it resembles it. It can be customized later if designs are provided.